### DISCUSSION TITLE
Improvements to the floating cart required

### DISCUSSION THREAD

**Discussion by Darksidetad** [addonify/addonify-floating-cart#292] [Ideas]
The size of the product title text is always in the same size, it would be nice if one could choose the size of the title.

The text size of the quantity x price is always in the same small size, it would be nice if one could choose the size of this text.

It would be nice if they could add a button to hide the button that says have a coupon.

The border of the quantity box should be hidden, it looks ugly or unnecessary in many cases one should be able to choose the thickness and if it says 0 it should disappear.

The size of the featured image of the product or variation should be able to change the size as it comes by default and takes up much space, if people add many products the scroller is very long, it would also be good that the image can be changed from square to circular using a radius.

The X button to remove the product that is above the featured image would be good to be a trash can, and that the fill or background of the circle can be changed.

Also in the floating cart icon, a blue box appears where the quantities that are added to the cart would be good to be able to edit that box, and also the radius and change it from square to round.

The title should be just the title, the variations should go below the separate title, and should be something like this:

**Product title 1**
**Color:** Color1, Color2, Color3, Color4
**Size:** Size1, Size2, Size3, Size4,
**Unit Price:** 00.00

---------------------------------------------

**Product title 2**
**Color:** Color1, Color2, Color3, Color4
**Size:** Size1, Size2, Size3, Size4,
**Unit Price:** 00.00

Making these modifications saves space in the "Cart width (unit: px)" and the size could be reduced to 400px or 350px, since large texts, large images, large spacing, force to stretch that area on the sides and also the scroller becomes long.

**Comment by h1dd3nsn1p3r** [comment]
Hi @Darksidetad 

I would like to start by thanking you for all the feedback & suggestions. 

I have moved your issue to a discussion as most of the points can be addressed with the feature update. We have acknowledged your feedback & we will start discussing this topic after the new year 2024 break.

We will provide the feedback here. Please feel free to join this discussion later.

**Reply by Darksidetad** [reply]
Hi @h1dd3nsn1p3r 
I would like to know how the development of this improvement is going, do you need screenshots...? You guys are really the best, you do a great job, for now there is no one that can match the level you have, this is love for design.
